device/arista/x86_64-arista_common/platform_asic

```diff
@@ -0,0 +1 @@
+broadcom
```

Comment:
i do not know what this is. this is common, it should be included in both broadcom and broadcom-dnx